AUTHORS.md

```diff
@@ -92,7 +96,9 @@ Max Sagebaum
 Michele Gaffuri
 Mickael Philit
 Mladen Banovic
+Nat-1
 Nicola Fonzi
+Nijso
```

Comment:
```suggestion
Nijso Beishuizen
```

Reply:
@bigfooted you have to decide whether how the world shall remember you, Bigfooted, Nijso (Beishuizen)